opendatahub-io/odh-manifests
# Apply notebook digest as environment variables on imagestreams

Apply notebook digest as environment variables on imagestreams.

Fix for: https://github.com/opendatahub-io/odh-manifests/issues/819
## Description


## How Has This Been Tested?

- Deploy ODH with enabled the notebook-images
- Login to odh-dashboard and navigate into notebooks server spawner
- Ensure that the notebooks selection is in place
- Try to spawn a notebook 

## Merge criteria:
